packages/ui/primitives/document-dropzone.tsx

```diff
@@ -10,6 +10,8 @@ import { megabytesToBytes } from '@documenso/lib/universal/unit-convertions';
 import { cn } from '../lib/utils';
 import { Card, CardContent } from './card';
 
+import {APP_DOCUMENT_UPLOAD_SIZE_LIMIT} from'@documenso/lib/constants/app';
```

Comment:
This should be causing linting issues, is your workspace configured correctly?

I also noticed some files in this PR are having final whitespace is being removed, which shouldn't happen if the environment is correctly configured

Reply:
I need help in this issue!

Reply:
Try restart vscode first if you haven't yet done so

When you go to this file, it should show up as linting errors

![image](https://github.com/documenso/documenso/assets/20962767/b449d6cb-95ce-48b1-bc8d-afd639860287)

Have a look in the vscode `problems` tab to see if it's working correctly

Also have a look bottom right of vscode to see if prettier is working, it should show up and have a check next to it

![image](https://github.com/documenso/documenso/assets/20962767/a715d8ef-1fc6-43fd-b003-57b6ff26a0fc)